### PR TITLE
Luca UI

### PR DESCRIPTION
**Pull request by luca020400**

### CODE REVIEW

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -232,78 +201,60 @@
...
-                android:id="@+id/constraintLayout"
-                android:layout_width="135dp"
-                android:layout_height="50dp"
-                android:background="@drawable/layout_camera_mode"
-                app:layout_constraintBottom_toTopOf="@+id/flipCameraButton"
-                app:layout_constraintEnd_toEndOf="parent"
-                app:layout_constraintHorizontal_bias="0.5"
-                app:layout_constraintStart_toStartOf="parent"
-                app:layout_constraintTop_toTopOf="parent">
+    <com.google.android.material.chip.ChipGroup
```
Comment (luca020400) [inline app/src/main/res/layout/activity_main.xml]: This layout doesn't let me evenly align the children, guess we'll have to use a plain old linear layout + our own items ( MaterialButtons? )

**Comment by luca020400** [comment]
Current layout

**Comment by luca020400** [comment]
Current UI


It's relatively good, I'm using the wrong kind of button as they match closely the mock-up but we should just make our own style later on ( wrong colour is also coming from them as they use an attribute we haven't set )

**Comment by luca020400** [comment]
Current UI gets us closer to the mock-up

Top bar items needs to be moved in a collapsable at the bottom sheet for ease of access
Buttons aren't using a proper style, that's the one that looks similar to mock-up without requiring customisation, we'll have to make our own style based off the standard one
Our current theme doesn't cover all the colours material3 uses, that's why we have purple buttons, we'll have to define our own colour scheme in a follow-up

**Comment by luca020400** [comment]
Can this go in?

**Comment by luk1337** [comment]
I'd skip these two for now:
image

**Comment by luca020400** [comment]
Agree on preview under cutouts, not sure why the other
Ship it!

**Comment by SebaUbuntu** [comment]
Is this PR still relevant?

**Comment by luk1337** [comment]
Not anymore i think.

**Pull request closed without merging**